v2/types.go

```diff
@@ -117,17 +117,30 @@ type ServiceInstanceSchema struct {
 // ServiceBindingSchema represents a plan's schemas for the parameters
 // accepted for binding creation.
 type ServiceBindingSchema struct {
-	Create *InputParametersSchema `json:"create,omitempty"`
+	Create *RequestResponseSchema `json:"create,omitempty"`
 }
 
 // InputParametersSchema requires a client API version >=2.13.
 //
 // InputParametersSchema represents a schema for input parameters for creation or
 // update of an API resource.
 type InputParametersSchema struct {
+	// The schema definition for the input parameters. Each input parameter
+	// is expressed as a property within a JSON object.
 	Parameters interface{} `json:"parameters,omitempty"`
 }
 
+// RequestResponseSchema requires a client API version >=2.14.
+//
+// RequestResponseSchema contains a schema for input parameters for creation or
+// update of an API resource, and a schema for the credentials returned by the
+// broker
+type RequestResponseSchema struct {
+	InputParametersSchema
+	// The schema definition for the broker's response to the bind request.
+	Response interface{} `json:"response,omitempty"`
```

Comment:
Perhaps call the object ResponseSchema?

Reply:
You mean rename the field `Response` to `ResponseSchema`? Not sure if that makes sense - the field name is aligned with the `Parameters` field in `InputParametersSchema` and matches the JSON field name. 

But what I would love is if I could rename `InputParametersSchema` to `RequestSchema`. Then, we'd have `RequestSchema.Parameters` and `RequestResponseSchema.Response`.

Reply:
I am sorry, I was 1000% mistaken. I was reading it as the object type Response, but it is of type interface{}. 

What you have is great.